### ISSUE TITLE
Changed Files

### ISSUE THREAD

**Issue by anteksiler** [ericmatthys/grunt-changelog#56]
Hi there,
I was unable to see an option to add a list of changed files under each commit. Is this possible?

**Comment by ericmatthys** [comment]
No, this is not currently possible. The task only looks at commit messages, not changed files.
